service-manual/user-centred-design/resources/patterns/feedback-pages.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: design-pattern
+title: Feedback pages
+breadcrumbs:
+  -
+    title: Home
+    url: /service-manual/index.html
+  -
+    title: User-centred design
+    url: /service-manual/user-centred-design/index.html
+  -
+    title: Design patterns
+    url: /service-manual/user-centred-design/resources/patterns/index.html
+---
+
+All digital services that government provides must have a Feedback page on GOV.UK. Every service must link to their feedback page to collect user satisfaction and feedback.
```

Comment:
Are they "feedback pages" or "Feedback pages"? The capitalisation of the first letter differs a few times in this file.